meson.build

```diff
@@ -533,8 +533,8 @@ configure_file(input : 'dbus/org.freedesktop.ratbag1.conf.in',
 
 #### tools ####
 
-python = import('python3')
-py3 = python.find_python()
+pymod = import('python')
+py3 = pymod.find_installation()
```

Comment:
you could escape this with a version check of meson itself and then make it dependent on 0.48 to do the right thing

Reply:
I haven't done this intentionally but since you prefer that solution I'll change the PR